tests/benchmark/codes/README.md

```diff
@@ -0,0 +1,15 @@
+# Guide to benchmark codes
+
+The source codes included in this directory can be used to reproduce all benchmark files used for the CCL unit tests. Several of these tests are based on the same software (e.g. the CosmoMAD library), but the output of that software was compared against other codes (i.e. CCL has not just been compared to CosmoMAD or any one particular package, but to several of them).
+
+* **distances_bm.py** : this script generates the comoving distances used in tests distances:model_1-5. To run it (and several of the other scripts below) you need to download and install the cosmomad library and python wrapper (https://github.com/damonge/CosmoMAD).
+* **distances_hiz_bm.py** : generates the high-redshift distances used in tests distances_hiz:model_1-3
+* **growth_bm.py** and **growth_hiz_bm.py** : same as distances_bm and distances_hiz_bm for the growth functions used in tests growth:model_1-5 and growth_hiz:model_1-3
+* **bbks_bm.py** and **ehpk_bm.py** : generates the benchmark BBKS and Eisenstein&Hu power spectra used in bbks:model_1-3 and eh:model_1.
+* **sigmaM_bm.py** : generates the benchmark sigma(M) values used in sigmam:model_1-3.
+* **mfunc_bm.py** : generates the mass function predictions used in massfunc:model_1.
+* **cl_corr_bm** : this folder contains the software needed to reproduce the benchmark files for angular power spectra and correlation functions used in tests cls:histo, cls:analytic, corrs:analytic_bessel and corrs:analytic_fftlog. The C code within that folder (limberjack) should be first compiled before running the script run_all.py.
```

Comment:
There doesn't seem to be a Makefile for this?

Reply:
added